pkg/controller/jobs/job/job_multikueue_adapter.go

```diff
@@ -49,6 +50,12 @@ func (b *multikueueAdapter) SyncJob(ctx context.Context, localClient client.Clie
 		return err
 	}
 
+	log := ctrl.LoggerFrom(ctx)
+	if ptr.Deref(localJob.Spec.Suspend, false) { // Ensure the job is resumed before updating its status; otherwise, it will fail when patching the spec.
```

Comment:
I think we should check it after the remote is created.

Reply:
exactly, changing it here is blind change, let's only modify if remote has started

Reply:
Done.